src/lando/treestatus/views.py

```diff
@@ -1,26 +1,24 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 import functools
 import logging
-from dataclasses import (
-    asdict,
-    dataclass,
-)
-from enum import Enum
+from datetime import datetime
 from typing import (
     Any,
     Callable,
+    Generic,
     Optional,
+    TypeVar,
 )
 
-import sqlalchemy as sa
-from connexion import ProblemException
-from flask import g
-from landoapi import auth
-from landoapi.cache import cache
-from landoapi.models.treestatus import (
+from django.core.cache import cache
```

Comment:
Do we actually have a cache set-up for django? https://docs.djangoproject.com/en/5.2/topics/cache/#:~:text=Your%20cache%20preference%20goes%20in%20the%20CACHES%20setting%20in%20your%20settings%20file%2E I see no `CACHE` entry in the `settings` https://github.com/mozilla-conduit/lando/blob/b823e62b9ef87b188b21b9db62537420482131ef/src/lando/settings.py, only in the `test_settings` https://github.com/mozilla-conduit/lando/blob/b823e62b9ef87b188b21b9db62537420482131ef/src/lando/test_settings.py#L19-L23

Reply:
The `treestatus.yml` file has a comment that says "This endpoint is cached heavily and is safe to call frequently to verify the status of a tree", so it seems we should be doing some form of caching to maintain that expectation going forward. The local filesystem cache is likely acceptable for this workload.

We previously had a Redis instance available for use in Lando, we could hook that up again if the performance of the local filesystem cache is poor.